src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -986,6 +982,10 @@ public abstract partial class Enum : System.ValueType, System.IComparable, Syste
         public static object ToObject(System.Type enumType, ushort value) { return default(object); }
         public override string ToString() { return default(string); }
         public string ToString(string format) { return default(string); }
+        [System.ObsoleteAttribute("The provider argument is not used. Please use ToString().")]
```

Comment:
We're making them implicit to be compat with desktop/Xamarin, as we've done many other places.